src/virtual_memory.cpp

```diff
@@ -99,7 +99,17 @@ void* allocMemoryPages(std::size_t bytes) {
 	#else
 		#define RESERVED_FLAGS 0
 	#endif
-	mem = mmap(nullptr, bytes, PAGE_READWRITE | RESERVED_FLAGS, MAP_ANONYMOUS | MAP_PRIVATE, -1, 0);
+	#ifdef __APPLE__
+		#include <TargetConditionals.h>
```

Comment:
Usually the '#' must be the first character on the line. If you want to indent, use something like "#    ifdef xxxx" instead

Reply:
Updated.

Reply:
@hyc That has not been the case since ANSI C AFAIK.

See for example http://www.open-std.org/jtc1/sc22/wg14/www/docs/n1124.pdf page 146 and onward.

Reply:
@selsta I prefer if you restore the original indentation for readability.

Reply:
This looks worse with no indentation at all. Might as well go back to your original indentation.

Reply:
done